### PR TITLE
Update geoserver to 2.18.1

### PR DESCRIPTION
**Pull request by meksor**
Update the geoserver by using a stripped down version of the newest https://github.com/kartoza/docker-geoserver image.

### CODE REVIEW

File: IIASA_TLDR.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ - Extensions will be downloaded when present in `build_data/stable_plugins.txt` during the build step. A full list of available can be found in `build_data/stable_plugins_available.txt`
+
+ - Extensions can be enabled by editing `.env` (`STABLE_EXTENSIONS=plugin1,plugin2`)
+   Downloaded extensions will be installed when creating the container (no need to rebuild the image, just `docker-compose up`).
+
+ - The data directory location inside the container can be adjusted with `GEOSERVER_DATA_DIR`.
+   It will always be exposed on the host system under `./geoserver_data` (for data imports).
+
+ - Tomcat will be routed to Port `:8600` by docker.
+
```
Comment (peterkolp) [inline IIASA_TLDR.md]: port needs to changed for prod (ene-geoserver-hotspots.iiasa.ac.at) 8222
and test (ece-geoserver-hotspots-test.iiasa.ac.at) 8223 geoservers running on hpg1002

File: docker-compose.yml [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
       volumes:
-        - /opt/ene-geoserver-hotspots.iiasa.ac.at/geoserver_data_dir:/opt/geoserver/data_dir
+        - ./geoserver_data:${GEOSERVER_DATA_DIR}
       ports:
-        - "8222:8080"
-      restart: always
-      env_file:
-        - docker-env/geoserver.env
-      container_name: ene_geoserver
+        - "8600:8080"
```
Comment (peterkolp) [inline docker-compose.yml]: ports for IIASA/ECE geoservesr are 8222 and 8223

File: clustering/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+version: '2.1'
+
+
+
+services:
+   master:
+      image: kartoza/geoserver:2.18.0
+      volumes:
```
Comment (peterkolp) [inline clustering/docker-compose.yml:8]: image: ene-docker.iiasa.ac.at/...:${GEOSERVER_VERSION_TAG}

File: clustering/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        - TOGGLE_SLAVE=true
+      restart: on-failure
+      healthcheck:
+        test: curl --fail -s http://localhost:8080/ || exit 1
+        interval: 1m30s
+        timeout: 10s
+        retries: 3
+   node:
+      image: kartoza/geoserver:2.18.0
+      volumes:
```
Comment (peterkolp) [inline clustering/docker-compose.yml:29]: see above

File: IIASA_TLDR.md (hunk 1 above)
Comment (meksor) [reply IIASA_TLDR.md]: Good point!
The host system port is now configurable using the `.env` file.

**Review by peterkolp — COMMENTED** [review]
What I've spotted is:

- image: specs 
- port #s (will need to be adjustet for hpg1002 prod/test geoserver isntances

sorry - too little knowlegde to judge the other changes
Not sure whether to pick Approve or Request changes

File: clustering/docker-compose.yml (hunk 3 above)
Comment (meksor) [reply clustering/docker-compose.yml:8]: I doubt we will be using clustering but for the sake of consistency I've changed this image name aswell.

File: docker-compose.yml (hunk 2 above)
Comment (meksor) [reply docker-compose.yml]: See above!

File: clustering/docker-compose.yml (hunk 4 above)
Comment (meksor) [reply clustering/docker-compose.yml:29]: -- " --

**Pull request merged**